index.js

```diff
@@ -23,7 +23,8 @@ exports.manifest = {
   replicate: 'duplex',
   request: 'sync',
   block: 'sync',
-  peerStatus: 'sync'
+  peerStatus: 'sync',
+  getClock: 'async'
```

Comment:
`getClock` in the manifest but the method exported is `clock`. Also we might need this added to `permissions`/`anonymous`?